biojava-structure/src/test/java/org/biojava/nbio/structure/basepairs/TestBasePairParameters.java

```diff
@@ -0,0 +1,79 @@
+package org.biojava.nbio.structure.basepairs;
+
+import org.biojava.nbio.structure.Structure;
+import org.biojava.nbio.structure.StructureException;
+import org.biojava.nbio.structure.StructureIO;
+import org.biojava.nbio.structure.basepairs.BasePairParameters;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by luke czapla on 7/21/17.
+ */
+public class TestBasePairParameters {
+
+    @Test
+    public void testBasePair() {
+
+        Structure structure;
+        try {
+            structure = StructureIO.getStructure("1KX5");
+        } catch (IOException|StructureException e) {
```

Comment:
Instead of catching the exceptions, add a `throws` declarations to the method, so that the `Test` status is `ERROR` instead of `FAIL` when the structure download gives an exception.